os/compression/compress_read.c

```diff
@@ -216,7 +218,7 @@ static off_t compress_lseek_block(int filfd, uint16_t binary_header_size, int bl
  *   Number of bytes read into read_buffer on Success
  *   Negative value on Failure
  ****************************************************************************/
-static off_t compress_read_block(int filfd, uint16_t binary_header_size, FAR uint8_t *buf, int block_number)
+off_t compress_read_block(int filfd, uint16_t binary_header_size, FAR uint8_t *buf, int block_number)
```

Comment:
In my opinion, we had not better make internal functions to public.

Reply:
These functions were made public as they are being called in the testcase file. Is there some other standard practice we use to handle this?

Reply:
Reverted the changes. Now these functions are no longer public functions.